test/unit/ui/marker.test.js

```diff
@@ -17,62 +17,201 @@ function createMap() {
 
 test('Marker', (t) => {
     t.test('constructor', (t) => {
-        const el = window.document.createElement('div');
+        const el = window.document.createElement(`mapboxgl-marker`);
         const marker = new Marker(el);
         t.ok(marker.getElement(), 'marker element is created');
         t.end();
     });
 
     t.test('default marker', (t) => {
```

Comment:
this test case was originally written to test that when no element is passed to Marker that it will use the default SVG, let's leave that test in place.